.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/tnolet/terraform-provider-checkly
```

Comment:
We may not actually need to set the working directory here; compare https://github.com/bitfield/uptimerobot/blob/master/.circleci/config.yml